_config.yml

```diff
@@ -59,6 +59,7 @@ tkui: "Telerik Kendo UI"
 tuin: "Telerik UI for NativeScript"
 
 # Telerik UI services
+ab-sab: "Code (AppBuilder)"
```

Comment:
This change is proposed by another colleague, both are ok for me. We can vote what is better :) .